### PR TITLE
[coffeescript/it] coffeescript in italian

### PR DESCRIPTION
**Pull request by kino90**

### CODE REVIEW

File: it-it/coffeescript-it.html.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+language: coffeescript
+contributors:
+  - ["Luca 'Kino' Maroni", "http://github.com/kino90"]
+  - ["Tenor Biel", "http://github.com/L8D"]
+  - ["Xavier Yao", "http://github.com/xavieryao"]
+filename: coffeescript-it.coffee
+lang: it-it
+---
+
+CoffeeScript è un piccolo linguaggio che compila direttamente nell'equivalente JavaScript, non c'è nessuna interpretazione a runtime. Come possibile successore di Javascript, CoffeeScript fa il suo meglio per restituire un codice leggibile, ben stampato e performante in ogni ambiente JavaScript.
```
Comment (geoffliu) [inline it-it/coffeescript-it.html.markdown]: Long line. Wrap at 80 characters

File: it-it/coffeescript-it.html.markdown [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Guarda anche [il sito di CoffeeScript](http://coffeescript.org/), che ha una guida completa a CoffeeScript.
+
+```coffeescript
+# CoffeeScript è un linguaggio hipster.
+# Segue le mode di alcuni linguaggi moderni.
+# Quindi i commenti sono come quelli di Ruby e Python, usano l'asterisco.
```
Comment (geoffliu) [inline it-it/coffeescript-it.html.markdown]: asterisco => cancelletto

**Comment by kino90** [comment]
Line 72 is still longer than 80 chars, but it's code.. is that all right or should i fix that too?

**Comment by geoffliu** [comment]
Eh, that's fine for now. Thanks!

**Pull request merged**